bgpd/bgp_vty.c

```diff
@@ -5649,9 +5649,11 @@ DEFUN (address_family_encapv6,
 
 DEFUN (address_family_evpn,
        address_family_evpn_cmd,
-       "address-family evpn",
+       "address-family <evpn|l2vpn evpn>",
```

Comment:
A commit has been pushed also to handle the change, meaning no evpn support, but l2vpn evpn.

From what you say, you mean evpn command hidden ? that means it is always possible to use address-family evpn, but nothing will be showed. right?